bugwarrior/command.py

```diff
@@ -15,6 +16,7 @@
 )
 
 import logging
+logging.basicConfig()
```

Comment:
I'm seeing the same behavior and I'm at a loss for why that is. All the other logging seems to work fine.